utils/data.py

```
@@ -0,0 +1,74 @@
+def load_json(filename: str) -> List[Dict[str, Any]]:
+    """
+    JSON 파일을 읽어 리스트 형태로 반환.
+    파일이 없거나 손상된 경우 빈 리스트 반환.
+    """
+    file_path = DATA_DIR / filename
+
+    if not file_path.exists():
+        return []
+
+    try:
+        with open(file_path, "r", encoding="utf-8") as f:
```

Ruff (0.14.13)

Unnecessary mode argument

Remove mode argument

(UP015)